export const getProducts=async(req,res)=>{
    try {
        
    } catch (error) {
        console.log(error);
        
    }
}